connector-runtime/connector-runtime-spring/src/test/java/io/camunda/connector/runtime/env/SpringEnvironmentSecretProviderTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.connector.runtime.env;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.env.MockEnvironment;
+
+public class SpringEnvironmentSecretProviderTest {
+
+  @Test
+  void shouldApplyPrefix() {
+    MockEnvironment env = new MockEnvironment();
+    env.setProperty("secrets.MY_TOTAL_SECRET", "beebop");
```

Comment:
Idea for better readability: What about using something that looks more like a prefix and not like our own secrets prefix like to not confuse the reader: `my_secret_prefix_` or similar.